hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java

```diff
@@ -194,7 +194,7 @@ protected String constructBuildString(BuildInfo buildInfo) {
     private void printNodeInfoInternal(BuildInfo buildInfo, String build) {
         systemLogger.info(getEditionString() + " " + buildInfo.getVersion()
                 + " (" + build + ") starting at " + node.getThisAddress());
-        systemLogger.info("Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.");
+        systemLogger.info("Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.");
```

Comment:
Shouldn't this be derived ?

Reply:
How do you mean derived? If the idea is to print the current year, I think we can't go this way due to copyright expiration reasons.

Reply:
Ok. So really my comment was "could this be derived" and the answer is "no".